src/sample/lecDashController.java

```diff
@@ -40,13 +57,38 @@ public void ContactUs(MouseEvent event) throws IOException {
     switchTo(event, "lecReport.fxml");
   }
 
+  public void CallUs() {
+    try {
+      FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("CallUs.fxml"));
+      Parent root1 = (Parent) fxmlLoader.load();
+      Stage stage = new Stage();
+      Image icon = new Image("sample/images/usm-ringlogo.png");
+      stage.getIcons().add(icon);
+      stage.setTitle("Contact Us");
+      stage.setResizable(false);
+      stage.setScene(new Scene(root1));
+      stage.show();
+    } catch (Exception e) {
+      System.out.println("Can't load new window");
+    }
+  }
```

Comment:
Put a video about this

Reply:
https://user-images.githubusercontent.com/93238154/151544125-62117047-edc6-4028-b41c-6af3a30cca3d.mp4